### ISSUE TITLE
"Fatal error: Cannot redeclare assetic_init()" when using Laravel workbench

### ISSUE THREAD

**Issue by dcelasun** [kriswallsmith/assetic#636]
Basically, if you have a package X in a Laravel workbench which depends on Assetic, and your main project depends on X, composer will try to load `src/functions.php` twice, resulting in a fatal error.

The obvious fix is to wrap the file in a

```
if (!function_exists('assetic_init'))
```

block. Other projects (like phpseclib) have encountered this same issue and they either used `function_exists` or [wrapped the functions in a class](https://github.com/phpseclib/phpseclib/pull/217).

I can send a PR to fix this if you are willing to merge it.

**Comment by leonardor** [comment]
Hi Kris!

It seems your fix from v1.2.1 doesn't do it alright.

I believe your code:

```php
if (function_exists('assetic_init')) {
    return;
}
```

should be:

```php
if (!function_exists('assetic_init')) {
    function assetic_init(AssetFactory $factory) {
        /* ... */
    }
}
```

because the autoloader will see anyway the defined function.

And that should be done with the other functions (assetic_javascripts, assetic_stylesheets, etc.)

**Comment by renaatdemuynck** [comment]
I'm having the same issue when trying to run PHPUnit from the command line. I patched the file as per leonardor's suggestion and that fixed the issue.
